src/state/reducers/scigateway.reducer.tsx

```diff
@@ -365,6 +366,10 @@ export function handleAuthProviderUpdate(
         provider = new JWTAuthProvider(payload.authUrl);
         break;
 
+      case 'ldapjwt':
```

Comment:
```suggestion
      case 'ldap-jwt':
```
The codecov warning here should also be fixed.